app/api/events.py

```diff
@@ -575,3 +579,9 @@ def clear_export_urls(event):
     event.xcal_url = None
     event.pentabarf_url = None
     save_to_db(event)
+
+
+@event_location_routes.route('/most-used-locations', methods=['GET'])
+def most_used_location():
+    locations = db.session.query(Event.location_name).group_by(Event.location_name).order_by(desc(func.count(Event.location_name))).all()
+    return jsonify({"locations": locations})
```

Comment:
@prateekj117 I am a bit skeptical about this one. can you please attach a sample response on this endpoint? meanwhile I will clone it and check on my local. Thanks!

Reply:
@uds5501 

> {
    "locations": [
        {
            "location_name": "Noida"
        },
        {
            "location_name": "ABC"
        },
        {
            "location_name": "Delhi"
        },
        {
            "location_name": "example1"
        }
    ]
}